src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -12,7 +12,7 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{5471BFE8-8071-466F-838E-5ADAA779E742}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <UseECMAKey Condition="'$(UseECMAKey)'==''">true</UseECMAKey>
```

Comment:
In Unix we are always defaulting `<UseECMAKey>false</UseECMAKey>` so this was fine before because both the ref and the implementation were not using it in non Windows for net46 build, but given that the ref for net46 is now a depproj, that one does have ECMA key signature, which causes GenFacades to fail if the implementation doesn't use the same key. For this reason, I'm basically forcing to use ECMA key always.